src/Application.php

```diff
@@ -472,8 +309,14 @@ public function route($path, $middleware = null, array $methods = null, $name =
         $this->checkForDuplicateRoute($path, $methods);
 
         if (! isset($route)) {
-            $methods = (null === $methods) ? Router\Route::HTTP_METHOD_ANY : $methods;
-            $route   = new Router\Route($path, $middleware, $methods, $name);
+            $methods    = (null === $methods) ? Router\Route::HTTP_METHOD_ANY : $methods;
```

Comment:
Useless parentheses around `null === $methods`.
Is that right that in case of empty array we will have an empty array there not `Router\Route::HTTP_METHOD_ANY`?